src/components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import "./ProductCard.css";
+import {IoCartOutline as CartOutline,IoHeartOutline as HeartOutline} from 'react-icons/io5';
+
+const ProductCard = ({productDetails}) => {
+
+    
+    // eslint-disable-next-line no-unused-vars
+    const {_id,title,categoryName,imgUrl,ratings,originalPrice,discountedPrice,discount,brand,inStock,wornBy}=productDetails
+    return (      
+        <div key={_id} className={`card card-vertical ${!inStock?"card-overlay":""}`}>
+            {!inStock?<div className="overlay-text-container">
+                <div className="card-overlay-text">OUT OF STOCK</div>
+            </div>:null}
```

Comment:
```
!inStock && <div className="overlay-text-container">
                          <div className="card-overlay-text">OUT OF STOCK</div>
                    </div>
```

Reply:
Got the understanding about it!!Thank you